cli/cmd/root.go

```diff
@@ -30,6 +32,28 @@ Find more information at:
 	logFile         *os.File
 )
 
+func getUserConfigDir() string {
```

Comment:
We should probably use https://github.com/adrg/xdg#xdg-base-directory to cover all use cases.
I think it might be better to use `XDG_CONFIG_HOME`. Then we won't need a CLI flag as we follow the `XDG` spec (i.e. if someone sets `XDG_CONFIG_HOME`)

Reply:
Nice! Will check this out.